image/image.go

```diff
@@ -102,6 +103,12 @@ func localSnaps(opts *Options) (*localInfos, error) {
 			info.Revision = snap.R(-1)
 			nameToPath[info.Name()] = snapName
 			local[snapName] = info
+
+			if si, err := snapasserts.DeriveSideInfo(snapName, &rdb{tsto}); err == nil {
```

Comment:
Thanks, I added that in 4ff47304206c8008c9dc565acfde90d6206bd710 - not sure its the best approach